test/node/debug.js

```diff
@@ -21,7 +21,7 @@
 
 'use strict';
 var assert = require('assert');
-var ObjectAssign = require('object.assign');
+var ObjectAssign = Object.assign;
```

Comment:
`engines: >=4` is required to be added.

Reply:
I've changed it to `node >= 12.6.0`, the current Node.js requirement of the nolyfill project, which should be enough.